### ISSUE TITLE
How can I reproduce the Cityscapes segmentation results

### ISSUE THREAD

**Issue by luuil** [hunto/DIST_KD#2]
I can't find the source code for distilling a segmentation model, would you please release it.

**Comment by hunto** [comment]
Hi @luuil ,

Sorry for the delay of releasing the segmentation code. Our code is mostly based on the code in the [CIRKD repo](https://github.com/winycg/CIRKD.git).

We will release our segmentation code this week (within 1~2 days if everything goes well).

Thanks,
Tao

**Comment by hunto** [comment]
> Hi @luuil ,
> 
> Sorry for the delay of releasing the segmentation code. Our code is mostly based on the code in the [CIRKD repo](https://github.com/winycg/CIRKD.git).
> 
> We will release our segmentation code this week (within 1~2 days if everything goes well).
> 
> Thanks, Tao

Hi @luuil ,

We have released the segmentation code. You can check the [segmentation](https://github.com/hunto/DIST_KD/tree/main/segmentation) folder in this repo for instructions.

If you have any question about the code or our paper, please feel free to contact us.

Best,
Tao

**Comment by luuil** [comment]
Thank you for releasing so quickly.

By the way, I have another question. 

As you mentioned in chapter `Appendix-A.5-Limitations` of your paper

> it would be less-effective on binary classification task

And I am working on a binary mask segmentation task(that is, for a input image, the goal of the task is to segment the fore&back ground) 😢.

What do you suggest for this?

**Comment by hunto** [comment]
> 

The binary classification tasks are often easily to be converged, and thus the prediction scores are very close to 0 or 1 (like one-hot labels). As a result, the teacher's logits have less relation information to be transferred than the challenging multi-task classification.

However, you can still try DIST and see if it is better than the conventional KD loss :)

**Comment by luuil** [comment]
Thanks for the suggestion, I'll give it a try.
